.github/workflows/build-image.yaml

```diff
@@ -59,6 +59,20 @@ jobs:
           done
         shell: bash
 
+      - name: Check
+        run: |
```

Comment:
We may need to make this step conditional to support non-linux builds. We could either make it opt-out or compare the runs_on input to ubuntu.

Reply:
Yes, sounds good.

Can we do a similar check for Windows too?

Reply:
Yeah, I have it on my todo to add a test and dev sku of the Windows build so I will add an analogous test as part of that. Filed #23 against myself.